_conferences/devfest-czech-republic-2018.md

```diff
@@ -0,0 +1,12 @@
+---
+name: "DevFest CZ 2018"
+website: https://2018.devfest.cz/
+location: Prague, CZ
+
+date_start: 2018-11-09
+date_end:   2018-11-10
+
+cfp_start: 2018-07-10
+cfp_end:   2018-08-26
+cfp_site: bit.ly/DevFestCZ18-CfP
```

Comment:
This probably needs an `http://` or `https://` to work properly on the site.

Reply:
https:// added